src/swsssdk/dbconnector.py

```diff
@@ -9,182 +9,324 @@
 # FIXME: Convert to metaclasses when Py2 support is removed. Metaclasses have unique interfaces to Python2/Python3.
 
 class SonicDBConfig(object):
+    SONIC_DB_BASE_DIR = "/var/run/redis/"
     SONIC_DB_CONFIG_FILE = "/var/run/redis/sonic-db/database_config.json"
+    SONIC_DB_GLOBAL_CONFIG_FILE = "/var/run/redis/sonic-db/database_global.json"
     _sonic_db_config_init = False
     _sonic_db_config = {}
 
     @staticmethod
-    def load_sonic_db_config(sonic_db_file_path=SONIC_DB_CONFIG_FILE):
+    def load_sonic_db_config(sonic_db_file_path=SONIC_DB_GLOBAL_CONFIG_FILE):
         """
-        Get multiple database config from the database_config.json
+        Parse and load the database config json files
+        Check if either of SONIC_DB_GLOBAL_CONFIG_FILE or SONIC_DB_CONFIG_FILE is present, unless the 
+        'sonic_db_file_path' is explicitly passed as parameter.
         """
         if SonicDBConfig._sonic_db_config_init == True:
             return
 
         try:
-            if os.path.isfile(sonic_db_file_path) == False:
-                msg = "'{}' is not found, it is not expected in production devices!!".format(sonic_db_file_path)
-                logger.warning(msg)
-                sonic_db_file_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'config', 'database_config.json')
-            with open(sonic_db_file_path, "r") as read_file:
-                SonicDBConfig._sonic_db_config = json.load(read_file)
+            if os.path.isfile(sonic_db_file_path) ==  True:
+                with open(sonic_db_file_path, "r") as read_file:
+                    all_dbs = json.load(read_file)
+                    for entry in all_dbs['INCLUDES']:
+                        if 'namespace' not in entry.keys():
+                            namespace = 'default'
+                        else: 
+                            namespace = entry['namespace']
+
+                        db_include_file = SonicDBConfig.SONIC_DB_BASE_DIR+entry['include']
```

Comment:
>entry['include'] [](start = 74, length = 16)

We need to support `include` relative path relative to `sonic_db_file_path`. Not relative to SONIC_DB_BASE_DIR.

Reply:
Updated.